en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md

```diff
@@ -101,8 +101,8 @@ As explained in the above section, you can create any user with a custom role to
 </tr>
 <tr class="even">
 <td>list apis</td>
+<td><strong>API Create</strong> or <strong>API Publish</strong> or <strong>API Subscribe</strong></td>
```

Comment:
For the list APIs call, we call the Publisher REST API. Hence I think API Subscribe is not required as it deals with the Dev Portal related operations.